tests/conf/i18n/translatehelpervisitor.js

```diff
@@ -0,0 +1,65 @@
+const TranslateHelperVisitor = require('../../../conf/i18n/translatehelpervisitor');
+const Translator = require('../../../conf/i18n/translator');
+const Handlebars = require('handlebars');
+const _ = require('lodash');
+
+async function createTranslator () {
+  const locale = 'fr';
+  return Translator.create(locale, [], { [locale]: { translation: {
+    'Hello': 'Bonjour',
+    'result': 'résultat',
+    'result_plural': 'résultats'
+  } } });
+}
+
+describe('TranslateHelperVisitor usage', () => {
+  it('static translation', async () => {
+    const template = `{{ translate phrase='Hello' }}`;
+    const ast = Handlebars.parse(template);
+    const translator = await createTranslator();
+
+    new TranslateHelperVisitor(translator).accept(ast);
+    const indexOfFirstStatement = 0;
+    const translatedValue = ast.body[indexOfFirstStatement].value;
+
+    expect(translatedValue).toEqual('Bonjour');
+  });
+
+  it('plural translation', async () => {
+    const template = `{{ translate phrase='result' pluralForm='results' count='resultCount' }}`;
```

Comment:
Should count for plurization be a number?

Reply:
It should be a variable because we don't know the value of count until runtime. I realized that I have it as a string and not a variable here , so I just pushed a commit which fixes that